.github/workflows/ci.yaml

```diff
@@ -952,6 +952,70 @@ jobs:
           name: oracle_18_integration_test_reports_${{ matrix.mode.label }}
           path: core/build/reports/tests/integrationTestJdbc
 
+  integration-test-for-jdbc-oracle-19:
```

Comment:
Since we've added this, we can remove `integration-test-for-jdbc-oracle-18`.

Reply:
Ok, it's revised in https://github.com/scalar-labs/scalardb/pull/2061/commits/d4c15a05ba3450c35c028249aef12403b9e35ed2

Reply:
Do we keep Oracle 18 on the CI for already released versions? Do you think about keeping Oracle 18 on the [Supported database](https://scalardb.scalar-labs.com/docs/latest/scalardb-supported-databases/) documentation or replacing it with Oracle 19 ?

Reply:
I think we can delete Oracle 18 from the supported database documentation. @feeblefakie What do you think?

Reply:
@brfrn169 I think so. The updated version is in my PR, so let me handle this.